app/routes/datasets/queries.py

```diff
@@ -85,6 +86,10 @@
 # Special suffixes to do an extra area density calculation on the raster data set.
 AREA_DENSITY_RASTER_SUFFIXES = ["_ha-1", "_ha_yr-1"]
 
+# Datasets that require admin privileges to do a query. (Extra protection on
+# commercial datasets which shouldn't be downloaded in any way.)
+PROTECTED_QUERY_DATASETS = ["licensed_wdpa_protected_areas"]
```

Comment:
I'm not sure how I feel about hardcoding this. If we need this very soon I wouldn't temporarily, but long term, I feel like we should make a general options to restrict access. 

@manukala6 curious, are there any things not downloadable that it's ok to query? Maybe that flag should just restrict viewing anything but metadata.

Reply:
It sounds from both of your comments that you are OK with doing this for now, so I'll go ahead with this check-in.  I had discussed with other folks, and they were OK as well, given that this is just a single dataset and we're not sure if we will need to protect any other datasets in this way.  A more general solution would require adding a whole new 'restricted_query' attribute to the data API DB schema.

We talked about restricting queries for all versions marked with is_downloadable as false.  However, there are already 8 different datasets with is_downloadable as false, including the public WDPA dataset.  So, I think we would definitely be affecting some current users if we restricted queries to all datasets versions with is_downloadable as FALSE.